docs/src/rules/no-restricted-imports.md

```diff
@@ -152,6 +152,20 @@ Specifying `"default"` string inside the `importNames` array will restrict the d
 }]
 ```
 
+Or like this if you need to restrict all imports from a module, except specified allowed ones.
```

Comment:
Can we keep the new options allowImportNames and allowImportNamePattern in the separate sections with their configurations and correct/incorrect examples under both paths and patterns as other options.

Reply:
@Tanujkanti4441 Thank you for your review. This is updated now.